docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -11,60 +11,66 @@ Usage
 
 .. code-block:: cppkokkos
 
-    Kokkos::RangePolicy<>(begin, end, args...)
-    Kokkos::RangePolicy<ARGS>(begin, end, args...)
-    Kokkos::RangePolicy<>(Space(), begin, end, args...)
-    Kokkos::RangePolicy<ARGS>(Space(), begin, end, args...)
+    Kokkos::RangePolicy<...>(begin, end)
```

Comment:
Are possible template parameter arguments shown below, i.e., `template < etc.`?  Assuming I've interpreted this correctly, please list template parameters in the "Parameters" section, which should be preceded by an "Interface" section (see PR#385).  Given that RangePolicy and ExecutionPolicy are coupled, you might also consider linking to [ExecutionPolicy](https://kokkos.github.io/kokkos-core-wiki/API/core/policies/ExecutionPolicyConcept.html?highlight=executionpolicy) in this entry